ansible/edge.yml

```diff
@@ -1,8 +1,8 @@
 ---
 # This playbook deploys Openwhisk Edge servers.
 # The edge is usually populated with NGINX serving as proxy.
-# The CLI also gets built and published for downloading from NGINX.  
-# SDKs for blackbox and iOS get published to NGINX also.
+# The CLI also gets built and published for downloading from NGINX.
+# SDKs for blackbox and iOS get published to NGINX also ( )
```

Comment:
What is `( )`?

Reply:
Text editor artifact.  Maybe a leftover from my not being sure where the SDKs come from.